src/number_theory/number_field.lean

```diff
@@ -43,6 +43,21 @@ class number_field (K : Type*) [field K] : Prop :=
 open function
 open_locale classical big_operators
 
+/-- `ℤ` is not a field. -/
+lemma int.not_field : ¬ is_field ℤ :=
```

Comment:
This is quite a weird statement - it doesn't just say "there can be no division operator on the integers", but makes the stronger statement "even if you redefine `*, +, 1, 0` to nonsense, you still can't make the integers a field" aka "every type of the same cardinality of the integers is not a field".